### DISCUSSION TITLE
How to get document by ID?

### DISCUSSION THREAD

**Discussion by d47081** [manticoresoftware/manticoresearch-php#192] [Q&A] [answered]
Is there any way to get row with it ID, not by attribute filter in search query?

e.g. `$row = $index->get(ID)`

Thanks!

**Comment by Nick-S-2018** [comment]
No, currently it's impossible to get a row in this way. If you're interested in such feature, please make a feature request for it.

**Reply by d47081** [reply, marked as answer]
Hm, are you sure? just found `getDocumentById` in

`vendor/manticoresoftware/manticoresearch-php/src/Manticoresearch/Index.php`

**Reply by Nick-S-2018** [reply]
Yes, I'm sorry for the  misleading answer. If you need just an alias for the attribute filter by 'id', you can use this function.
